assets/scss/6-components/text-input/text-input.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <label for="firstName" class="l-display-block has-xxxs-btm-marg t-size-s">
+    <strong>First name</strong>
+  </label>
+  <input id="firstName" name="firstName" type="text" value="Cosmo Kramer" class="c-text-input is-invalid l-display-block l-width-full has-text-gray-dark has-xxxs-btm-marg" aria-invalid="true">
+  <ul>
+    <li class="has-text-error has-xxxs-btm-marg t-size-xs">
+      First validation error
+    </li>
+    <li class="has-text-error t-size-xs">
+      Second validation error
+    </li>
+  </ul>
+</div>
```

Comment:
What was your thinking on only showing the invalid state in the demo? 
I like having that since it's the less straightforward variation, but it might be helpful to also see the more common state. I think it's fine for the html to have more than one component if it's helpful documentation.

Reply:
I added the valid state.